client/hosting/overview/components/hosting-overview.tsx

```diff
@@ -20,8 +21,19 @@ import './style.scss';
 const HostingOverview: FC = () => {
 	const site = useSelector( getSelectedSite );
 
-	if ( site && isMigrationInProgress( site ) && getMigrationStatus( site ) === 'pending' ) {
-		return <MigrationOverview site={ site } />;
+	if ( site ) {
+		const migrationType = getMigrationType( site );
+		const migrationStatus = getMigrationStatus( site );
+
+		if (
+			isMigrationInProgress( site ) &&
+			// TODO: Remove the following checks when we support all migration types.
```

Comment:
When we support the pending/difm, we can check it only for `isMigrationInProgress( site )` because we'll have variations for all the states under progress.